browser/css/device-desktop.css

```diff
@@ -31,7 +31,7 @@ button:not(.ui-corner-all):not(.button-primary):hover {
 	cursor: pointer;
 	color: var(--color-text-darker) !important;
 	background-color: var(--color-background-lighter) !important;
-	border: 1px solid var(--color-border-darker);
+	border: 0px solid var(--color-border-darker);
```

Comment:
why are we removing the border?

Reply:
Thanks for checking @pedropintosilva . It flickers the UI on hover, so i needed to disable it.